packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.controller.js

```diff
@@ -76,6 +77,14 @@ export default class TelecomPackMigrationOffersCtrl {
       }
     });
 
+    if (optionName === 'modem') {
```

Comment:
because you use *modem* more than one place (l80, l191),
plz, consider to create a const variable or add it to constant file and import it.
could be something `const MODEME_OPTION_NAME = 'modem';`